### ISSUE TITLE
Snow should be able to install system packages on /nix/var/nix/profiles/default

### ISSUE THREAD

**Issue by dnkmmr69420** [snowflakelinux/snow#10]
If snow isn't running on nixos, it should allow you to use the -s flag and have packages installed in the nix default profile.

**Comment by dnkmmr69420** [comment]
or maybe there can be a new flag -S for installing packages to `/nix/var/nix/profiles/default`

**Comment by jakehamilton** [comment]
I think snow currently installs to the default profile for the user? @vlinkz can correct me if I'm wrong here.

https://github.com/snowflakelinux/snow/blob/main/src/profile/install.rs#L35

@dnkmmr69420 what is your use case here? Is there a particular problem you are trying to solve?

**Comment by dnkmmr69420** [comment]
`/nix/var/nix/profiles/default` is a global nix profile that is available for all users. I would like to use this on my silverblue machine. Root owns that profile but any user can run programs from it.

**Comment by jakehamilton** [comment]
@dnkmmr69420 is this because you're trying to support multiple users on the same system? It would help to understand why this change would be useful.

**Comment by dnkmmr69420** [comment]
I was saying that there is no `/etc/nixos/configuration.nix` on silverblue. I was saying that snow should install in the root profile when ran with the `-d` option

**Comment by dnkmmr69420** [comment]
it is useful if you want to install something for all users

**Comment by jakehamilton** [comment]
Right, so the problem is that, currently, you cannot use Snow to manage packages for multiple users. Is that correct @dnkmmr69420?

**Comment by jakehamilton** [comment]
Out of curiosity, do you share your machine with others? Or do you use multiple accounts for another reason?

**Comment by dnkmmr69420** [comment]
I do use multiple accounts on my computer. It is also because there is no configuration.nix on silverblue too

**Comment by vlinkz** [comment]
I think this makes sense. Especially since system profiles are not/rarely used on NixOS given the existence of a configuration.nix file.

**Comment by dnkmmr69420** [comment]
And there is also some things on nixos that you want it system wide but don't want it to be managed by configuration.nix
